src/js/utils/use-isomorphic-layout-effect.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-restricted-imports */
+import React, { useEffect } from 'react';
+
+/**
+ * A substitute for React's useLayoutEffect, which does not generate warnings on
+ * SSR. It is named useLayoutEffect so that all eslint rules applying to the
+ * original useLayoutEffect would also apply to it.
+ * This solution was suggested by Alex Reardon.
+ * @see https://medium.com/@alexandereardon/uselayouteffect-and-ssr-192986cdcf7a
+ * @see https://github.com/grommet/grommet/issues/4765
+ */
+export const useLayoutEffect =
```

Comment:
What do you think about exporting this as default? For me it makes more sense based on the module name.

Reply:
I like the non-default export because it makes usage similar to that of the original useLayoutEffect (which is typically imported by name).
However I don't see any harm in adding a default export too, will add.